### DISCUSSION TITLE
请教各位大佬如何通过源地址自动分流？

### DISCUSSION THREAD

**Discussion by EricZhou05** [vernesong/OpenClash#4150] [Q&A]
![image](https://github.com/user-attachments/assets/de699a26-b47c-4e38-99c5-5e7dca4d24e7)

如图，内网有两个客户端，如何让其中一个客户端经过clash内核分流访问国外，另外一个客户端直接绕过内核不使用openclash直连国内呢？

**Comment by fejeIaqbhbv** [comment]
OpenClash的插件设置里不是有黑白名单吗？

**Comment by EricZhou05** [comment]
谢谢，本地DNS劫持使用防火墙转发，然后用黑白名单过滤即可

**Reply by EricZhou05** [reply]
但是使用防火墙转发会丢失一些功能

[如何使广告拦截配合OpenClash使用](https://github.com/Aethersailor/Custom_OpenClash_Rules/issues/78)

我想到一个好方法：
直接覆写设置-规则附加自定义规则让源ip直连

`- SRC-IP-CIDR,192.168.2.185/32,DIRECT`
或者
`- "SRC-IP-CIDR,192.168.2.185/32,\U0001F3AF 全球直连,no-resolve"`

就是DHCP难解决，可能要给设备绑静态IP
